docs/reference/settings/ml-settings.asciidoc

```diff
@@ -119,7 +135,7 @@ concurrently be in the `opening` state on each node. Typically, jobs spend a
 small amount of time in this state before they move to `open` state. Jobs that
 must restore large models when they are opening spend more time in the `opening`
 state. When the {operator-feature} is enabled, this setting can be updated only 
-by operator users. Defaults to `2`.
+by operator users. The minimum value is `0`. Defaults to `2`.
```

Comment:
```suggestion
by operator users. Defaults to `2`.
```

I think you should temporarily remove the minimum value of 0.  Then this PR can be merged and when we fix #70064 we can add a sentence here about what 0 means after the fix.

Reply:
Thanks, that sounds like a good plan to me!